docs/source/cpp/compute.rst

```diff
@@ -1639,17 +1639,19 @@ in the respective option classes.
-+=====================+============+=====================================+==================+==============================+========+
-| list_element        | Binary     | List-like (Arg 0), Integral (Arg 1) | List value type  |                              | \(1)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
-| list_flatten        | Unary      | List-like                           | List value type  |                              | \(2)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
-| list_parent_indices | Unary      | List-like                           | Int64            |                              | \(3)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
-| struct_field        | Unary      | Struct or Union                     | Computed         | :struct:`StructFieldOptions` | \(4)   |
-+---------------------+------------+-------------------------------------+------------------+------------------------------+--------+
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| Function name       | Arity      | Input types                         | Output type      | Options class                   | Notes  |
++=====================+============+=====================================+==================+=================================+========+
+| list_element        | Binary     | List-like (Arg 0), Integral (Arg 1) | List value type  |                                 | \(1)   |
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| list_flatten        | Unary      | List-like                           | List value type  |                                 | \(2)   |
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| list_parent_indices | Unary      | List-like                           | Int64            |                                 | \(3)   |
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| struct_field        | Unary      | Struct or Union                     | Computed         | :struct:`StructFieldOptions`    | \(4)   |
++---------------------+------------+-------------------------------------+------------------+---------------------------------+--------+
+| map_array_lookup    | Unary      | MapArray                            | Computed         | :struct:`MapArrayLookupOptions` | \(5)   |
```

Comment:
Please keep these tables alphabetically sorted (sorry, you may have to renumber some items)

Reply:
```suggestion
| map_array_lookup    | Unary      | Map                                 | Computed         | :struct:`MapArrayLookupOptions` | \(5)   |
```